### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by InfernoCycle** [InfernoCycle/YouTubeDownloader#2] [Q&A]
- Same as 2.0.0 but supports Windows 8 and up.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfernoCycle/TrueYTtoMP3/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by weiwei187242** [comment]
加油呀。感谢更新。我用了，很好用。
